dnf/base.py

```diff
@@ -1218,7 +1222,7 @@ def _translate_comps_pkg_types(pkg_types):
                 ret |= enum
         return ret
 
-    def group_install(self, grp, pkg_types, exclude=None):
+    def group_install(self, grp_id, pkg_types, exclude=None):
```

Comment:
having `base.group_*` to accept string makes it consistent with `base.install/remove/...` but we cannot change the API. At least Anaconda is using it. Can you add there a few compatibility lines i.e.:
```python
# compat: erase in 2.0.0
if isinstance(grp_id, Comps):
    grp_id = grp_id.id
```